File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        
        idx = 0
        p_l = []
        
        
        for num in nums:
            if num < pivot:
                p_l.append(num)
            idx += 1
            
        for num in nums:
            if num == pivot:
                p_l.append(pivot)
            idx += 1
            
        for num in nums:
            if num > pivot:
                p_l.append(num)
            idx += 1
                
        return p_l
                

File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.go
func pivotArray(nums []int, pivot int) []int {
    var idx int
    res := make([]int, len(nums))
    for i := range nums {
        if nums[i]<pivot {
            res[idx]=nums[i]
            idx++
        } 
    }
    
    for i := range nums {
        if nums[i]==pivot {
            res[idx]=pivot
            idx++
        } 
    }
    
    for i := range nums {
        if nums[i]>pivot {
            res[idx]=nums[i]
            idx++
        } 
    }
    
    
    return res
}